packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -270,18 +281,22 @@ TradingHubHeader.propTypes = {
     switchToCRAccount: PropTypes.func,
 };
 
-export default connect(({ client, notifications, ui, traders_hub }) => ({
+export default connect(({ client, common, notifications, ui, menu, traders_hub }) => ({
     header_extension: ui.header_extension,
+    is_app_disabled: ui.is_app_disabled,
     is_dark_mode: ui.is_dark_mode_on,
     is_eu_country: client.is_eu_country,
     is_eu: client.is_eu,
     is_logged_in: client.is_logged_in,
     is_mt5_allowed: client.is_mt5_allowed,
     is_notifications_visible: notifications.is_notifications_visible,
+    is_route_modal_on: ui.is_route_modal_on,
     modal_data: traders_hub.modal_data,
     notifications_count: notifications.notifications.length,
     toggleNotifications: notifications.toggleNotificationsModal,
     loginid: client.loginid,
+    menu_items: menu.extensions,
```

Comment:
@suisin-deriv `MenuStore` doesn't exist anymore, Not sure how the build is passing 🤔

Reply:
Aite will remove that 👍